logic/src/commonTest/kotlin/com/wire/kalium/logic/feature/call/GetAllCallsUseCaseTest.kt

```diff
@@ -0,0 +1,63 @@
+package com.wire.kalium.logic.feature.call
+
+import com.wire.kalium.logic.data.call.CallRepository
+import com.wire.kalium.logic.data.id.ConversationId
+import com.wire.kalium.logic.feature.call.usecase.GetAllCallsUseCase
+import com.wire.kalium.logic.sync.SyncManager
+import io.mockative.Mock
+import io.mockative.classOf
+import io.mockative.given
+import io.mockative.mock
+import kotlinx.coroutines.flow.MutableStateFlow
+import kotlinx.coroutines.flow.first
+import kotlinx.coroutines.test.runTest
+import kotlin.test.BeforeTest
+import kotlin.test.Test
+import kotlin.test.assertEquals
+
+class GetAllCallsUseCaseTest {
+
+    @Mock
+    private val callRepository = mock(classOf<CallRepository>())
+
+    @Mock
+    private val syncManager = mock(classOf<SyncManager>())
+
+    private lateinit var getAllCallsUseCase: GetAllCallsUseCase
+
+    @BeforeTest
+    fun setUp() {
+        getAllCallsUseCase = GetAllCallsUseCase(
+            callRepository = callRepository,
+            syncManager = syncManager
+        )
+    }
+
+    @Test
+    fun givenAFlowOfCalls_whenUseCaseInvoked_thenReturnThatFlow() = runTest {
+        given(syncManager).coroutine { waitForSlowSyncToComplete() }
+            .then { println("DONE") }
```

Comment:
```suggestion
            .thenReturn(Unit)
```

Let's not make our console dirty when testing, shall we?

Reply:
Agree